jvb/src/main/resources/reference.conf

```diff
@@ -289,6 +289,21 @@ videobridge {
     }
   }
 
+  loudest {
+      # The number of current loudest speakers to route audio packets for.
+      numLoudest = 3
```

Comment:
Can we have a way to disable the new behavior and default to it?

Reply:
off by default now.